user-guide/Reference/Skyline_DataMiner_Folder/More_information_on_certain_files_and_folders/MaintenanceSettings_xml.md

```diff
@@ -57,6 +57,7 @@ This is an example of a *MaintenanceSettings.xml* file:
   <Trending>
     <EDCurves></EDCurves>
     <SDCurves></SDCurves>
+    <!-- CAUTION: SLDataGateway crashes on startup when MaintenanceSettings.xml contains <TimeSpan1DayRecords window="0"> -->
```

Comment:
Wouldn't it be better if we updated the example to a different value instead of adding this caution?

Reply:
Fine by me, howeever:
- this is not enbaled by default, so user should not assume this is added by default
- it should then document that "0" is not a valid value

Reply:
I've updated your pull request. Could you check if everything still looks OK to you?